app/models/actor.rb

```diff
@@ -0,0 +1,12 @@
+class Actor < ApplicationRecord
+  has_many :movie_actors
+  has_many :movies, through: :movie_actors
+
+  def self.ordered_by_age
```

Comment:
The US actually is looking for this data to be for the actors in a *specific movie* so these calculations would make more sense in the Movie model, where we can call the actor attribute and it will just be a collection of actors associated to that movie:
`actors.order(:age)`

Reply:
This is true for both the model methods here -- they should be instance methods in Movie